foundations/html_css/css-foundations/the-cascade.md

```diff
@@ -134,7 +134,7 @@ Here both rule 1 and rule 2 have the same specificity. Rule 1 uses a chaining se
 }
 ~~~
 
-This example shows the same thing. Even though rule 2 is using a child combinator (`>`), this does not change the specificity value. Both rules still have two classes so they have the same specificity values.
+This example shows the same thing. Even though rule 2 is using a child combinator (`>`), which will be discussed in more detail in the lesson [Advanced Selectors](https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-advanced-selectors), this does not change the specificity value. Both rules still have two classes so they have the same specificity values.
```

Comment:
I do not like to link to lessons in the future, rather just point out that they will learn it in the future. However, if we are referring to previous a lesson, I like to include a link. Can you please either update this change or revert it back to how it way?

Reply:
I do not think this rule makes much sense, but if you prefer it this way, I'm reverting the change.

Reply:
The reason behind this is because we want people to follow our lessons in order. If we link to lessons in the future, a person is prone to do that entire future lesson and is likely to struggle to understand everything. This is very intentional decision from the maintainer team.